src/app/components/lists/list-overview-item/list-overview-item.component.html

```diff
@@ -1,25 +1,27 @@
-<div class="row shadow mr-1 ml-1 mt-3 mt-sm-0 animated" [ngClass]="{'fadeIn':canShowToolTip}" id="main">
+<div class="modal-header cardTopRow p-0">
+    <button class="close" (click)="deleteModal.open()">
+        <i class="material-icons deleteListBtn">
+            highlight_off
+        </i>
+    </button>
+</div>
+<div class="cardHeadband text-center shadow pointer" [routerLink]="'/wishlists/' + list._id">{{list.name ? list.name : 'Untitled List'}}</div><div class="row animated" [ngClass]="{'fadeIn':canShowToolTip}" id="main">
```

Comment:
This should be on two lines and also the row sticks out from the card. This is why I dont like the card class because rows look weird. Maybe bootstrap has an answer to this?